GoogleForCreators/web-stories-wp
# Dashboard: Added animations to Entertainment pages 3 through 6

## Summary

This PR adds animations to pages 3 - 6 of the Entertainment template.

These animations are first iterations, missing animation features we'll want for polish are noted in this doc:
https://docs.google.com/spreadsheets/d/1iJ8Adqs4zGWteMoxVCZ2MM4LAza0LQmtPUX6I9qdshY/edit#gid=21496487

## To-do

Apply animations to the remaining Entertainment pages.

## User-facing changes

None.  Animations are disabled by default.

## Testing Instructions

1.) Go to `Dashboard.php` and set `enableAnimation` to `true`.
2.) Go to "Explore Templates" and hover over the Entertainment template, you should see the following animations play out:
![ent-page-36](https://user-images.githubusercontent.com/40646372/84925528-e3a79500-b07e-11ea-86f7-79f4916c208a.gif)

---

## Animations Close up

![ent_page3](https://user-images.githubusercontent.com/40646372/84925967-8cee8b00-b07f-11ea-80d0-cd6bebac4aa3.gif)

![ent_page4](https://user-images.githubusercontent.com/40646372/84925972-8f50e500-b07f-11ea-97dc-d78ca9787945.gif)

![ent_page5](https://user-images.githubusercontent.com/40646372/84925980-91b33f00-b07f-11ea-8b30-90dcd7fe4436.gif)

![ent_page6](https://user-images.githubusercontent.com/40646372/84925983-937d0280-b07f-11ea-9d75-2c6e6d5237e1.gif)
